Add Kong Mesh tooltips (#4145)
Kong/docs.konghq.com: src/mesh/index.md

<td class="no-mobile"></td>
</tr>
<tr>
<td>
Roles and permissions (RBAC)
{% info_tooltip %}
This enterprise capability allows us to manage complex RBAC rules to allow - or deny - access to Kong Mesh policies and functions in a sophisticated and fine grained way. Works across multi-zone and multi-mesh natively.
{% endinfo_tooltip %}
</td>
<td><i class="fa fa-times"></i></td>
<td><i class="fa fa-check"></i></td>
</tr>
<tr>
<td>
Audit Logs
{% info_tooltip %}

> This enterprise capability allows us to store and fetch auditing logs for operations that have been performed on the cluster and - when used with RBAC - allows us to have full visibility into how the system is being governed and configured by the users.

Reviewer: ```suggestion
Allows you to store and fetch auditing logs for operations that were performed on the cluster. When used with RBAC, it allows us to have full visibility into how the system is being governed and configured by the users.
```

<td><i class="fa fa-check"></i></td>
</tr>
<tr>
<td>
Kubernetes Operator
{% info_tooltip %}

> This enterprise capability allows us to enormously simplify the operational cost of running Kong Mesh on Kubernetes by providing a native Kubernetes operator to fully manage the deployment, upgrades and rollouts of a cluster on Kubernetes.

Reviewer: ```suggestion
Allows you to simplify the operational cost of running Kong Mesh on Kubernetes by providing a native Kubernetes operator to fully manage the deployment, upgrades, and roll outs of a cluster on Kubernetes.
```

<td><i class="fa fa-check"></i></td>
</tr>
<tr>
<td>
Native AWS ECS Controller
{% info_tooltip %}

> This enterprise capability allows us to natively support AWS ECS workloads with a built-in controller that automatically integrates ECS workloads within one or more service meshes powered by Kong Mesh, simplifying the expansion of service mesh in the cloud.

Reviewer: ```suggestion
Allows you to natively support AWS ECS workloads with a built-in controller that automatically integrates ECS workloads within one or more service meshes powered by Kong Mesh. This simplifies the expansion of service mesh in the cloud.
```

{% endinfo_tooltip %}
</td>
<td><i class="fa fa-times"></i></td>
<td><i class="fa fa-check"></i></td>
</tr>
<tr>
<td>
Windows Distributions
{% info_tooltip %}

> This enterprise capability allows us to natively support Microsoft Windows workloads into our service meshes, allowing to us further expand the reach of Kong Mesh across every workload in the organization.

Reviewer: ```suggestion
Allows you to natively support Microsoft Windows workloads in service meshes, allowing you to further expand the reach of Kong Mesh across every workload in your organization.
```

{% endinfo_tooltip %}
</td>
<td><i class="fa fa-times"></i></td>
<td><i class="fa fa-check"></i></td>
</tr>
<tr>
<td>
UBI Federal Distributions
{% info_tooltip %}

> This enterprise capability provides officially supported distributions based on the Red Hat Universal Base Images (UBI).

Reviewer: ```suggestion
Provides officially supported distributions based on the Red Hat Universal Base Images (UBI).
```

<td class="no-mobile"></td>
</tr>
<tr>
<td>
Enterprise Support and SLA
{% info_tooltip %}

> With Kong Mesh we provide 24/7/265 enterprise support with different SLAs, powered by Kong's global customer success and technical support team across all world regions, and access to a vast network of partners for local language support as well. A must to have for enterprise mission-critical deployments.

Reviewer: ```suggestion
With Kong Mesh, we provide 24/7/365 enterprise support with different SLAs, powered by Kong's global customer success and technical support team across all world regions. This also provides access to a vast network of partners for local language support as well. This is recommended for enterprise mission-critical deployments.
```

{% endinfo_tooltip %}
</td>
<td><i class="fa fa-times"></i></td>
<td><i class="fa fa-check"></i></td>
</tr>
<tr>
<td>
Customer Success Packages
{% info_tooltip %}

> With Kong Mesh we provide access to our implementation and training programs to accelerate the rollout of a service mesh across every team, and to properly train and educate the organization on how to effectively drive business outcomes with the product.

Reviewer: ```suggestion
With Kong Mesh, we provide access to our implementation and training programs to accelerate the roll out of a service mesh across every team, and to properly train and educate the organization on how to effectively drive business outcomes with the product.
```

{% endinfo_tooltip %}
</td>
<td><i class="fa fa-times"></i></td>
<td><i class="fa fa-check"></i></td>
</tr>
<tr>
<td>
Envoy Support
{% info_tooltip %}

> With Kong Mesh we provide access to the Envoy contributors at Kong to further expand the capabilities of the underlying data plane proxy technology (Envoy) with features that are not currently available in upstream Envoy, in order to remove road blocks and cater to unique enterprise requirements and use-cases.

Reviewer: ```suggestion
With Kong Mesh, we provide access to the Envoy contributors at Kong to further expand the capabilities of the underlying data plane proxy technology (Envoy) with features that are not currently available in upstream Envoy. This can be used to remove road blocks and cater to unique enterprise requirements and use-cases.
```
